### PR TITLE
Create ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60003](https://cumulocity.atlassian.net/browse/MTM-60003)
Original PR: [6875](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6875)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix for widgets not rendering when scroll is enabled (#6822) [GRAFT][release/cd] (#6875)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed widgets not rendering when scroll is enabled
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60003
+version: 1020.14.12
+---
+Fix for widgets not rendering when scroll is enabled (#6822) [GRAFT][release/cd] (#6875)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some cases, widgets were not properly rendering when scrolling was enabled on a page. This issue has been resolved and widgets will now consistently render as expected, even when scroll is turned on. The fix ensures a smoother user experience when viewing and interacting with widget content on pages that allow scrolling.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md (hunk 1 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:3]:
```suggestion
title: Fixed new widgets not rendering when scroll is enabled
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md (hunk 2 above)
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:17]:
```suggestion
In some cases, new widgets were not properly rendering when scrolling was enabled on a page. This issue has been resolved and widgets will now consistently render as expected, even when scroll is turned on.
```

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60003
+version: 1020.14.12
+---
+In some cases, new widgets were not properly rendering when scrolling was enabled on a page. This issue has been resolved and widgets will now consistently render as expected, even when scroll is turned on.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:17]:
```suggestion
In some cases, new widgets were not properly rendering when scrolling was enabled on a page. This issue has been resolved and widgets will now consistently render as expected when scroll is turned on.
```

File: content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed new widgets not rendering when scroll is enabled
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-14-12-fix-for-widgets-not-rendering-when-scroll-is-enabled-6822-graft.md:3]:
```suggestion
title: New widgets render properly when scroll is enabled
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**